vault/helm/vault/templates/vault-configmap.yaml

```diff
@@ -75,12 +114,12 @@ data:
     mount_accessor=$(vault auth list | awk '{ if ($1 == "oidc/") { print $3 } }')
     echo "  mount accessor: $mount_accessor"
 
-    echo "Writing 'admins' group"
-    group_id=$(vault write identity/group name=admins policies=admin type=external | awk '{ if ($1 == "id") { print $2 } }')
-    echo "  group id: $group_id"
+    echo "Writing 'admin' user"
```

Comment:
@DavidSpek I thought you asked to me to introduce an "admin group", which is what I did. Could you please explain to me why you changed that to a single user? Thanks.

Reply:
A way to expose the admin group configuration to the user was never added, which would mean that unintended users could become admin in vault installations where they shouldn't have had these privileges. Down the road we can create a more configurable way to add users/groups and policies.